pages/docs/frontend-modules/releasing.en-US.mdx

```diff
@@ -56,10 +76,37 @@ This command:
 - Tells yarn to sort the packages before running the commands so that they run on the packages in topological order (i.e., packages that depend on other packages get run later).
 - Tells yarn to exclude versioning for the selected monorepo so that the package.json in the root directory is unaffected. The root `package.json` does not require a version for publishing purposes because it is never published.
 
-Once this process completes, you should see a diff in your editor that includes a version bump for all the packages in your monorepo. Run `yarn` or `yarn install` to update your `yarn.lock` file. Commit these changes with the title `(chore) Release vx.x.x` where x.x.x is a placeholder for the version number. See an example of [this commit](https://github.com/openmrs/openmrs-esm-patient-chart/pull/1279) that bumps `Patient Chart` to `v5.0.0`.
+#### Non-monorepos
+
+For non-monorepos, you can use the `version` command directly. For example, if you want to bump the version of a package named `@openmrs/esm-patient-management` to a `major` release, you would run:
+
+```sh
+yarn version major --workspaces --scope @openmrs/esm-patient-management
+```
+
+<Callout type="info">
+**Libraries using legacy versions of yarn**
+
+Some of our projects use legacy versions of yarn (yarn v1). The [Angular form engine](https://github.com/openmrs/openmrs-ngx-formentry) is an example of such a project. To release the Angular form engine, you would run the following command:
+
+```sh
+yarn version --new-version <major|minor|patch> --no-git-tag-version
+```
+
+Be sure to follow the instructions listed in the [Cutting a release docs](https://github.com/openmrs/openmrs-ngx-formentry#cutting-a-new-release) to avoid getting caught out.
+
+</Callout>
+
+### Post-bump steps
+
+Once you're done versioning, you should see a diff in your editor that includes a version bump for all the packages in your repository. Run `yarn` or `yarn install` to update your `yarn.lock` file.
+
+Commit these changes with the title `(chore) Release vx.x.x` where x.x.x is a placeholder for the version number. See an example of [this commit](https://github.com/openmrs/openmrs-esm-patient-chart/pull/1279) that bumps `Patient Chart` to `v5.0.0`.
```

Comment:
Minor, probably obvious, but should we mention that this should be done in a new branch, and to make a PR to main?